packages/angular/build/src/tools/angular/transformers/jit-bootstrap-transformer.ts

```diff
@@ -32,7 +32,7 @@ export function replaceBootstrap(
             bootstrapImport = nodeFactory.createImportDeclaration(
               undefined,
               nodeFactory.createImportClause(
-                false,
```

Comment:
We also need to make similar changes in the framework, but we can't until we've switched to TS 5.9 internally. As far as I can tell, this code isn't synced into g3.